src/netconfig.cpp

```diff
@@ -203,15 +203,9 @@ static void cmdDns(Dbus& bus, Arguments& args)
 
     const char* nextArg = args.peek();
     const bool isStatic = nextArg && strcmp(nextArg, "static") == 0;
-    const char* property;
     if (isStatic)
```

Comment:
I think we may remove ii, it resolves BBMC-313.

Reply:
As I wrote in the commit message, I left it for backward compatibility.
It does not affect on the command execute any more.
But if you sure that backward compatibility is not required I will remove it completely.

Reply:
We have no any released firmware with netconfig, so backward compatibility is not required.
Moreover, QA asks to fix the "static" flag: https://j.yadro.com/browse/BBMC-313

Reply:
done